src/askui/models/anthropic/computer_agent.py

```diff
@@ -30,16 +35,23 @@ def __init__(
     def _create_message(
         self, messages: list[MessageParam], model_choice: str
     ) -> MessageParam:
+        _thinking: ThinkingConfigParam = ThinkingConfigDisabledParam(type="disabled")
+
+        if self._settings.thinking:
+            _thinking = ThinkingConfigEnabledParam(
+                type="enabled",
+                budget_tokens=self._settings.thinking.budget_tokens,
+            )
```

Comment:
Why not just use `pydantic.BaseModel`s of a similar structure inside the settings so that we don't have to do this extra transformation and can just pass it along? Makes it also easier for the user if already familiar with Anthropic.